src/dirac_cwl_proto/submission_models.py

```diff
@@ -65,6 +64,18 @@ def validate_hints(cls, values):
         return values
 
 
+class JobSubmissionModel(BaseJobModel):
+    """Job definition sent to the router."""
+
+    job_inputs: list[JobInputModel] | None = None
```

Comment:
Sorry, my fault, could you remove the `job_` part in the attribute (since we are already in a `Job<Something>` object).

```suggestion
    inputs: list[JobInputModel] | None = None
```